new_platforms/vscode-extension/README.md

```diff
@@ -142,12 +153,28 @@ data to Microsoft, you can set the `telemetry.enableTelemetry` setting to `false
 ## Known Issues
 
 * Building SGX enclaves is not currently supported.
+* On Linux, if you see errors in an Edge container build that involve `standard_init_linux.go:207`, you are likely cross-building a container.  Adding these packages solved the cross-build issue on Ubuntu 19.04:
+
+      sudo apt-get install -y qemu qemu-user-static qemu-user binfmt-support
+
+* We've had reports that downloading the SDK from git can be slow from within the extension, to work around any issue here, you can run these commands
```

Comment:
Consider removing "here", since it doesn't add any information.

Reply:
ok